### ISSUE TITLE
Remove the Reporting Rates feature

### ISSUE THREAD

**Issue by garethbowen** [medic/cht-core#6248]
**Describe the issue**
The Reporting Rates feature is as far as we can tell not used, it's probably not working any more, is a lot of technical debt, and a lot of code.

**Describe the improvement you'd like**
Check again if it's used as best as you can and then delete the feature as a breaking change in the next major.

**Describe alternatives you've considered**
We could separate it out as a distinct module so it doesn't affect users as much (as in #3869 ) but it'd be even better if it didn't exist at all.

**Comment by MaxDiz** [comment]
@garethbowen I'm not familiar with the reporting rates feature. Can you link to documentation or give me a quick summary so I can look into it a bit further?

**Comment by garethbowen** [comment]
There is no documentation. It can be enabled in the Analytics tab if you have some forms marked as periodic. Then administrators can track which facilities haven't reported in for that time period. Once upon a time it was used for tasks like stock monitoring where facilities had to SMS in their stock counts.

**Comment by garethbowen** [comment]
I'm investigating whether we can bring this forward to 3.11.0 so we can save converting the large reporting rates feature to Angular 10 only to drop it altogether in the next major.

In order for the feature to show the configuration must have a `kujua-reporting` array with a form code that matches one of the configured JSON forms. To determine if _any_ project has these configured I have searched through the medic-projects repo (except those in the `archive` folder), and every `medic` repo that starts with `config-`. I found many instances of `kujua-reporting` being defined but most were misconfigured or didn't have a matching JSON form, quite likely copied from the [default config](https://github.com/medic/cht-core/blob/master/config/default/app_settings.json#L40).

There were no instances for which the reporting rates feature would be shown.

**Comment by garethbowen** [comment]
[Forum post](https://forum.communityhealthtoolkit.org/t/feature-removal-proposal-reporting-rates/1053) proposing removing the feature.

**Comment by garethbowen** [comment]
Nobody has spoken up in defense of this feature so let's remove it. If someone wants it we can look at backporting and migrating it, or rebuilding it from scratch, but we shouldn't hold up the Angular work migrating a feature that nobody uses.

**Comment by latin-panda** [comment]
This is done in: https://github.com/medic/cht-core/pull/6765

**Comment by latin-panda** [comment]
This is fixed and merged in the [angular migration branch](https://github.com/medic/cht-core/tree/4750-angular-10-take-2). We'll move the ticket to AT once the migration branch is merged into master.

**Comment by garethbowen** [comment]
Removed!
